packages/core-typings/src/IUser.ts

```diff
@@ -164,7 +164,8 @@ export interface IRegisterUser extends IUser {
 }
 
 export const isRegisterUser = (user: IUser): user is IRegisterUser => user.username !== undefined && user.name !== undefined;
-export const isUserFederated = (user: Partial<IUser>): user is IUser => 'federated' in user && user.federated === true;
+export const isUserFederated = (user: Partial<IUser>): user is IUser =>
+	'federated' in user && user.federated === true && Boolean(user.username?.includes(':'));
```

Comment:
Is it really necessary to check by `:`?  checking by `federated` being true is not enough?

Reply:
No, it's not, I'll remove it, thanks for pointing it out.